bl/policy/policy_factory.go

```diff
@@ -72,7 +73,16 @@ func populateRules(policyRules []cliClient.Rule, customRules []*cliClient.Custom
 		customRule := getCustomRuleByIdentifier(customRules, rule.Identifier)
 
 		if customRule != nil {
-			rules = append(rules, RuleWithSchema{rule.Identifier, customRule.Name, customRule.Schema, rule.MessageOnFailure})
+			if customRule.Schema == nil {
+				schema := make(map[string]interface{})
+				err := json.Unmarshal([]byte(customRule.JsonSchema), &schema)
```

Comment:
please consider extracting this into a function, I think we use this several times.
`func jsonSchemaToStruct()`

we already have `func yamlToStruct()`

Reply:
1. `yamlToStruct` is not used in a general way, it's a function that is used with a specific struct so I don't even know why it's called `yamlToStruct`. 
2. Even though there is a yamlToStruct function we still use `yaml.Unmarshall` outside of this function, so that indicates on what happens with utils function in general so wrapping it in `jsonSchemaToStruct` function will not guarantee usage of this function in the future